tests/fsharp/optimize/tests_optimize.fs

```diff
@@ -0,0 +1,276 @@
+module ``FSharp-Tests-Optimize``
+
+open System
+open System.IO
+open NUnit.Framework
+
+open NUnitConf
+open PlatformHelpers
+open FSharpTestSuiteTypes
+
+let testContext = FSharpTestSuite.testContext
+
+module Analyses = 
+
+    let ``fsc >a 2>&1`` cfg dir = 
```

Comment:
This naming is too cryptic for me - one ``>`` I can handle, but not ``>a 2>&1``.  Please give it a long name?